reinforcement_learning/Lesson-4-Policy-Based-Learning-Method-draft.ipynb

```diff
@@ -0,0 +1,403 @@
+{
```

Comment:
<p>Rephrase: "...we have talk(ed)..." to "...we talked..."</p><p>Corrected typo: "...value-based methods...", "...policy-based methods.", and "...difference between value-based and policy-based methods."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BreakoutMentors/Data-Science-and-Machine-Learning/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>